monero-outreach-docs/translations/es/storyboard/tool_for_renewal_es.md

```diff
@@ -0,0 +1,78 @@
+
+### _Adquiriendo Monero_  
+
+**Monedero.** Para empezar a adquirir Monero y usarlo, debes tener un monedero de Monero. Existen muchas opciones de monederos y abarcan factores como la facilidad de uso, la seguridad y la portabilidad. Nosotros en Monero Outreach hemos creado una [guía sencilla para que aprendas más sobre Monero e inicies con un monedero](https://www.monerooutreach.org/cuentos/monedero-de-monero.html).  
+
+**Intercambio.** Si tienes una cuenta bancaria o ya eres dueño de otras criptomonedas, puedes usar una casa de cambio con buena reputación como [Binance.com](https://www.binance.com/es) o [Kraken.com](https://www.kraken.com/) para adquirir Monero. El código o ticker para el intercambio de Monero es XMR. Con un banco puedes transferir fondos directamente hacia la casa de cambio, comprar XMR y tranferir tus XMR a tu propio monedero. Ten en cuenta que algunas casas de cambio tienen emparejado a XMR con otras criptomonedas, a lo mejor, tendrás que comprar primero otra criptomoneda y despues canjearla por XMR. Encuentra casas de cambio que enlistan a Monero:  
+
+- [https://web.getmonero.org/es/community/merchants/#exchanges](https://web.getmonero.org/es/community/merchants/index.html#exchanges)  
+- [Recursos para Inversores de Monero](https://www.monerooutreach.org/investors/)  
+- [https://coincodex.com/crypto/monero/exchanges/](https://coincodex.com/crypto/monero/exchanges/)  
+
+**Comerciantes privados.*** Puedes comprar o vender Monero con comerciantes privados, en linea o en persona, usando [LocalMonero.co](https://localmonero.co/).  
+
+**Minar.** Cuando minas Monero, no solo tienes la oportunidad de ganar XMR al intercambiar poder de procesamiento de computo, sino que también ayudas a asegurar la red. Monero tiene un algoritmo innovador que vuelve accesible la minería con hardware común y puedes empezar a minar sin ninguna inversión adicional. Comienza a minar:  
+
+- [https://web.getmonero.org/es/get-started/mining/](https://web.getmonero.org/es/get-started/mining/)  
+- [Minar Monero para apoyar la red](https://www.monerooutreach.org/mejores_practicas_monero/mina-para-apoyar-la-red.html)  
+- [https://github.com/xmrig/xmrig](https://github.com/xmrig/xmrig)  
+
+**Crypto ATMs.** Puedes comprar XMR sin un banco utilizando dinero en efectivo o tarjetas de crédito en los cajeros ATMs de criptomonedas alrededor del mundo. Estos comunmente vienen con comisiones altas pero son convenientes. Para ubicar uno cerca de ti, usa un servicio como [CoinAMTRadar.com](https://coinatmradar.com/) y filtra el mapa por monedas (Coins): Monero (XMR) y operaciones (Operations): Comprar (Buy).  
+
+**Aceptando Monero (para negocios).** La comunidad global de desarrollarodes de Monero ha creado herramientas para empoderar a los negocios a aceptar XMR para sus productos y servicios. Corten de la ecuación al costoso intermediario y permitanle a sus clientes pagar fracciones de centavos por comisiones en transacciones para usted. Aprende más:  
```

Comment:
Empoderar lo cambiaría por "permitierles" o "capacitar"

Reply:
- "Corten de la ecuación" —- No seria mejor "eliminen/saquen de la ecuacion

Reply:
Yeah you're right. I made the changes. Fixed :)